ui/src/logs/actions/index.ts

```diff
@@ -51,12 +85,73 @@ export const setSource = (source: Source): SetSourceAction => ({
   },
 })
 
-export const setNamespace = (namespace: Namespace): SetNamespaceAction => ({
-  type: ActionTypes.SetNamespace,
-  payload: {
-    namespace,
-  },
-})
+export const executeHistogramQuery = () => async (
```

Comment:
what do you think about using the convention of suffixing this with `Async` since it's a thunk?